Agents/FenlandTrajectoryAgent/resources/ontop.obda

```diff
@@ -1,45 +1,169 @@
+source      SELECT "FHRSID", EXTRACT(YEAR FROM "Business_Establishment_End") AS year
+            FROM "fhr_cambridge_2025"
+
+mappingId       RatingPeriodEnd-2023
+target      ies4:ParticularPeriod/{FHRSID}_rating_end/{year} a ies4:ParticularPeriod ; ies4:iso8601PeriodRepresentation {Business_Establishment_End}^^xsd:dateTimeStamp .
+source      SELECT "FHRSID", "Business_Establishment_End", EXTRACT(YEAR FROM "Business_Establishment_End") AS year
+            FROM "fhr_cambridge_2023"
+
+mappingId       RatingPeriodEnd-2025
+target      ies4:ParticularPeriod/{FHRSID}_rating_end/{year} a ies4:ParticularPeriod ; ies4:iso8601PeriodRepresentation {Business_Establishment_End}^^xsd:dateTimeStamp .
+source      SELECT "FHRSID", "Business_Establishment_End", EXTRACT(YEAR FROM "Business_Establishment_End") AS year
+            FROM "fhr_cambridge_2025"
+
+mappingId       Rating_Details-2023
+target      fh:Rating/{FHRSID}/{year} a fh:Rating ; fh:hasValue "{RatingValue}"^^xsd:string .
+source      SELECT "FHRSID", "RatingValue", EXTRACT(YEAR FROM "Business_Establishment_Start") AS year 
+            FROM "fhr_cambridge_2023"
+
+mappingId       Rating_Details-2025
+target      fh:Rating/{FHRSID}/{year} a fh:Rating ; fh:hasValue "{RatingValue}"^^xsd:string .
+source      SELECT "FHRSID", "RatingValue", EXTRACT(YEAR FROM "Business_Establishment_Start") AS year
+            FROM "fhr_cambridge_2025"
+
+mappingId       LocalAuthorityMapping
+target      fh:BusinessEstablishment/{FHRSID} fh:hasLocalAuthority fh:LocalAuthority/{LocalAuthorityCode} .
+source      SELECT DISTINCT "FHRSID", "LocalAuthorityCode" FROM "fhr_cambridge_2025"
+
+mappingId       LocalAuthorityDetails
+target      fh:LocalAuthority/{LocalAuthorityCode} a fh:LocalAuthority ; fh:hasName "{LocalAuthorityName}"^^xsd:string ; fh:hasWebsite "{LocalAuthorityWebSite}"^^xsd:anyURI ; fh:hasEmailAddress "{LocalAuthorityEmailAddress}"^^xsd:string .
+source      SELECT DISTINCT "LocalAuthorityCode", "LocalAuthorityName", "LocalAuthorityWebSite", "LocalAuthorityEmailAddress" 
+            FROM "fhr_cambridge_2025"
+
+mappingId       GreenspaceCore
+target      gs:{id} a gs:Greenspace ; gs:hasFunction "{function}"^^xsd:string ; gs:hasFirstAlternativeName "{distName1}"^^xsd:string ; gs:hasSecondAlternativeName "{distName2}"^^xsd:string ; gs:hasThirdAlternativeName "{distName3}"^^xsd:string .
+source      SELECT "id", "function", "distName1", "distName2", "distName3"
+            FROM "GB_GreenspaceSite"
+
+mappingId       GreenspaceGeometry
+target      gs:{id} a geo:Feature ; geo:hasGeometry geo:Geometry{id} .
+source      SELECT "id"
+            FROM "GB_GreenspaceSite"
+
+mappingId       GS-Geometry-details
+target      geo:Geometry{id} geo:asWKT "{wkt}"^^geo:wktLiteral ; ex:hasWKB "{wkb_hex}"^^xsd:string ; geo:crs <http://www.opengis.net/def/crs/EPSG/0/27700> .
+source      SELECT "id", "wkb_geometry" AS "wkb_hex", ST_AsText("wkb_geometry") AS "wkt"
+            FROM "GB_GreenspaceSite"
```

Comment:
I don't think it is a good idea to obtain WKB (the binary string from PostGIS), the conventional way should be to process WKT literals

Reply:
Added a to-do item in commit 98718d68c6d53c847c71471c352d66216342f00b.